en/setup/dev_env_linux_arch.md

```diff
@@ -0,0 +1,48 @@
+# Development Environment on ArchLinux
+
+## Permissions
+
+The user needs to be added to the group "uucp":
```

Comment:
@jlecoeur @dagar This group needed to be set in the old version. Is it still required? Is this the right group?

I can't test this as I cannot get archlinux working. Also, I did get a message that sudo command is not supported on archlinux - thoughts?

Reply:
Yes it is still needed. uucp is the arch equivalent of dialout.

Weird for sudo. If you want to install an arch VM, I recommend Antergos. It has a GUI installer that takes care of everything